stylesheets/commons/BigMatch.less

```diff
@@ -795,6 +797,72 @@ ul.match-bm-lol-game-veto-overview-pick {
 	}
 }
 
+.match-bm-game-veto-overview-team-header {
+	border-bottom: 0.0625rem solid var( --clr-secondary-25 );
```

Comment:
Missing a dark theme variant

Reply:
Update and tested [here](https://liquipedia.net/dota2/Liquipedia:ID_XLA7bhOs3l_R01-M001)